src/views/search/search-conversation-list-item.tsx

```diff
@@ -123,14 +164,26 @@ const SearchListItem = ({ itemId, item, folderId, selected, selecting, toggle, a
 		},
 		[conversationStatus, dispatch, parent, item.id]
 	);
+
+	const msgToDisplayCount = useMemo(() => {
+		const result = !searchInTrash
+			? filter(item?.messages, (msg) => msg.parent !== FOLDERS.TRASH)?.length
+			: item?.messages?.length;
+		return messagesToRender.length !== 0 ? messagesToRender.length : result;
+	}, [item?.messages, messagesToRender.length, searchInTrash]);
+
+	const allMessagesInTrash =
```

Comment:
variable name doesn't match with the returned value

Reply:
please read it again